build.gradle.kts

```diff
@@ -0,0 +1,59 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+group = rootProject.group
```

Comment:
should we define group and version here as well?

```
group = "org.jetbrains.research.extractMethod"
version = "1.0"
```
it's a root project isn't it?